go.mod

```diff
@@ -75,6 +72,7 @@ require (
 	github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f // indirect
 	github.com/opencontainers/image-spec v1.1.0-rc4 // indirect
 	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
+	github.com/prometheus/client_model v0.6.1 // indirect
```

Comment:
we don't need this line; please remove it.

Reply:
ran `go mod tidy`

Reply:
(it's till there, something is using it, but not us directly)

Reply:
it's wired i did same and go mod tidy removed it 🤔

Reply:
yes from the first `require` block, and moved it into the second `require` block (to the `// indirect` ones)

Reply:
okey now it's gone.